### ISSUE TITLE
aws: decrypt-tls-assets.service is broken

### ISSUE THREAD

**Issue by colhom** [coreos/coreos-kubernetes#670]
```sh
sudo /opt/bin/decrypt-tls-assets 
Unable to locate credentials. You can configure credentials by running "aws configure".
```

The quick fix is to add `--net=host` to `docker run` so aws cli can reach AWS the metadata service. @pbx0 could this be related to the network plugin switch? 

Anyways, fixed by #666

**Comment by pbx0** [comment]
It might be a result of the docker0 bridge not being the bridge used by the pod network anymore which is related to the cni stuff.

**Comment by colhom** [comment]
Fixed in #666.
